pkg/cmd/pulumi/import.go

```diff
@@ -291,8 +299,13 @@ func newImportCmd() *cobra.Command {
 	var protectResources bool
 
 	cmd := &cobra.Command{
-		Use:   "import [type] [name] [id]",
-		Args:  cmdutil.MaximumNArgs(3),
+		Use: "import [type] [name] [id] [properties...]",
```

Comment:
I'm surprised that this is an argument, and not specified with a flag. 
1. Given the other flags, it makes more sense in my head to have it as a flag.
2. It locks in the `pulumi import` basic (flag-less) api until we break backwards compatibility. 
3. It renders a little funny in the help menu:
```
Usage:
  pulumi import [type] [name] [id] [properties...] [flags]
```

Reply:
Yeh that's a fair comment. Looking at what the args library supports it looks like we could have:
`--properties propa,propb,propc` / `-p propa,propb -p propc`
or
`--property propa --property propb --property propc` / `-p propa -p propb -p propc`

I think `--properties a,b,c` is probably the nicer choice?

Reply:
> I think --properties a,b,c is probably the nicer choice?

I would agree